### DISCUSSION TITLE
"U30" component

### DISCUSSION THREAD

**Discussion by Almenau14** [Severson-Group/AMDC-Hardware#242] [General]
Hi,
I send the gerbers to PCBWay and them answer me with this message: "The correct package for the listed Part Number [TPS22965DSGT] is indeed [WSON-8-EP(2x2)], not [SON50P200X80_HS-9N].Please confirm." (the "U30 component).
I studied the datasheet and looks corretclty. 
Another issue is that they do not have UJ2-BH-4-TH (CON13) in stock. Can this be substituted with a similar component?

![U30](https://github.com/user-attachments/assets/83a993e9-2dc2-4029-9125-5431ef23868e)

**Comment by npetersen2** [comment]
The `CON13` is a standard USB-B type connector, so can easily be substituted with a footprint compatible part. For example, this [part](https://www.digikey.com/en/products/detail/cui-devices/UJ2-BH-2-TH/6188079) is the same, but with different plating thickness. It should still work.

Regarding `U30`, I believe the footprint used on the `REV F` boards have worked well for assembly... I think it can still be assembled.

**Reply by Almenau14** [reply]
Hi, 
My question is whether component "U30" is the one listed in this link:
https://www.digikey.in/en/products/detail/texas-instruments/TPS22965DSGT/3661875

**Reply by codecubepi** [reply]
Yes, you and PCBWay are correct. It appears we have the wrong package/footprint for U30 in our design files. Thank you for the catch! Feel free to open an issue in the [AMDC-Hardware repository](https://github.com/Severson-Group/AMDC-Hardware/issues) to flag this as something to correct in a future revision. You can use the "Create issue from discussion" option to the right of the thread as well.

The part you've linked to above on DigiKey is correct 🙂.
